hack/prow_job_volume.py

```diff
@@ -0,0 +1,47 @@
+#!/bin/env python3
+"""This script query the Prometheus instance
+prometheus-prow-monitoring.apps.ci.l2s4.p1.openshiftapps.com
+to get the workload of Prow Job in the past 7 days and save it as a json file"""
+
+
+import argparse
+import json
+import logging
+import os
+import requests
+
+
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s:%(levelname)s:%(message)s"
+    )
+
+def main():
```

Comment:
Can we please make this proper in go and with tests and set up basic auth or something so this can be executed automatically in a job rather than "copy cookie from your webbrowser"?

Reply:
I have to admit that i see this coming.